data-complexity/fisher_discriminant_ratio.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+import numpy as np
+from dcm import dcm
+from sklearn.datasets import load_iris
```

Comment:
This dependency should be added to the [requirements.txt](https://github.com/jkarolczak/data-complexity-measures/blob/main/requirements.txt) and [environment.yml](https://github.com/jkarolczak/data-complexity-measures/blob/main/environment.yml)